fix(nx): wrap files inputted to prettier in quotes

ZachJW34 opened the pull request:
Files that contained parentheses caused 'yarn format' to fail unless the files passed to the
prettier cli are wrapped in quotes

fix #1473

FrozenPandaz commented on packages/workspace/src/command-line/format.ts:
```diff
@@ -96,17 +96,24 @@ function printError(command: string, e: any) {
-    });
+    execSync(
+      `node "${prettierPath()}" --write ${patterns
+        .map(pattern => (fileExists(pattern) ? `'${pattern}'` : pattern))
```
Why do we need to check if the file exists before wrapping it in quotes? It should be filtered out higher above on https://github.com/nrwl/nx/pull/1485/files#diff-f8549c910ef16fe4ad9fa75fd90c4005R47

ZachJW34 replied:
I originally had it as
```
.map(pattern => `'${pattern}'`
```
but the e2e command-line test failed. This was due to patterns also being wrapped in quotes. The `fileExists` check is to determine whether an element of the patterns input is a pattern e.g. "${prefix}/**/*.{${PRETTIER_EXTENSIONS.join(',')}}" or an actual file. The `fileExists` could be replaced by a `isFile` function that could do some regex to determine whether it s a pattern or a file. What do you think?

FrozenPandaz replied:
Right, so we can move this mapping up to https://github.com/nrwl/nx/blob/65ad548ed74f7903575138ed631cd5ff8bc7aacd/packages/workspace/src/command-line/format.ts#L59 which is the only time when it has actual file paths.

ZachJW34 replied:
Oh I see. I'll update the PR soon!

ZachJW34 commented:
Updated. I moved the mapping down to the return, and I only wrap the files if `libsAndApp` is false. Wrapping the files with quotes beforehand will cause `getPatternsFromApps` to incorrectly find touched projects.
